benchmarks/auth/python/Dockerfile.pypy

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
We should not include this dockerfile. 
We dont need the PyPy version of python at the moment. This was more for research but I think we should not release it. Its not necessary at the moment.